src/main/java/com/google/flourbot/api/CloudDocClient.java

```diff
@@ -4,7 +4,5 @@
 import java.security.GeneralSecurityException;
 
 public interface CloudDocClient {
-
-  // TODO: change exception handling to try catch. throw illegal state exception
```

Comment:
that's a good comment, we don't want to throw these.

Reply:
Added as issue so that I remember!